src/less/blog-v2/includes/latest-articles.less

```diff
@@ -21,6 +21,12 @@
     padding: @category-padding;
     background-color: @color-category-card-tag-background;
     color: @color-category-card-tag-font;
+    a {
```

Comment:
The spacing got kinda wonky in the .less files. Is this something that Prettier did?